tailor_management_system/urls.py

```diff
@@ -2,9 +2,13 @@
 
 from django.urls import path
 
-from tailor_management_system.views import dashboard_view
+from tailor_management_system.views.dashboard_view import dashboard_view
```

Comment:
i think we should keep two views files, 1. views.py(for simple views), 2. api_views.py (for api views)

Reply:
@Ayub-Khan we already have two separate view files for these, what do you mean exactly?

Reply:
I mean is that its not appropriate to create a seperate file for a single view. so we can make two generic files one for api views (api_views.py) and one for other views (views.py) and that dashboard_view should be a part of views.py